modular_skyrat/modules/busy_space/loremaster.dm

```diff
@@ -0,0 +1,11 @@
+var/datum/lore/loremaster/loremaster = new/datum/lore/loremaster
+
+/datum/lore/loremaster
```

Comment:
wtf we have lore

Reply:
here it comes baby 😎 

Most of it'll be kept in organizations.dm